lib/server.js

```diff
@@ -13,7 +14,7 @@ const weblog = require('webpack-log');
 
 const WebpackServeError = require('./WebpackServeError');
 
-module.exports = (options) => {
+module.exports = options => {
```

Comment:
this will fail linting, as will several other changes in this file. run `npm run lint` before the next commit.

Reply:
yea I tried running the linter but its giving me about 2000 failures:

```
  /* many more like this */
  50:1   error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style
  51:45  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style
  52:1   error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style
  53:11  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style
  54:3   error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

✖ 2273 problems (2273 errors, 0 warnings)
  2273 errors, 0 warnings potentially fixable with the `--fix` option.

```

Reply:
that looks like you have an environment (editor, etc) issue on your end there. perhaps your editor isn't recognizing or obeying the linting setup for the repo?

Reply:
I think git is messing with me since the files are indeed crlf (both my editor and eslint agree on that) formatted but git doesn't seem to think its a problem (unless i convert everything in which case it tells me im changing every file in the repo)